algorithms/appo/model_utils.py

```diff
@@ -257,6 +257,35 @@ def forward(self, obs_dict):
         return x
 
 
+class MujocoEncoder(EncoderBase):
```

Comment:
Don't introduce a new class. Just add another if in encode_subtype check in the existing MLP encoder. Because I see that this encoder is exactly the same as MLP encoder.